0020-day-two/scrum-of-scrums/readme.md

```diff
@@ -0,0 +1,23 @@
+# Scrum of Scrums
+
+For those that may be unfamiliar (until the content below is completed)... [Scrum of Scrums](https://www.agilealliance.org/glossary/scrum-of-scrums)

```

Comment:
...For those who are unfamiliar...

Reply:
Fixed in https://github.com/microsoft/code-with-engineering-playbook/pull/67/commits/004a8aa2afa9e43c184a0bd96b4361fe76d0cbf3.